github/__init__.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Stefan Schnyder
+
+"""
+Searches GitHub user repositories and opens the selected match in the browser
+Trigger with 'gh '
+Rebuild cache with 'gh rebuild cache'
+"""
+
+import os
+import json
+import keyring
+from albert import *
+from pathlib import Path
+import github3
+from rapidfuzz import fuzz
```

Comment:
albert provides a builtin matcher. pleease use it. do not search fuzzy by default. see an example https://github.com/albertlauncher/python/blob/53c4bdda1c4edcef226159deef4279739414fda0/jetbrains_projects/__init__.py#L229-L241

Reply:
Done. I took the code from the emoji plugin as an example instead to make the fuzzy search an option.